### PR TITLE
(NFC-ish) Multiple doc block cleanups

### PR DESCRIPTION
**Pull request by totten**
I ran https://github.com/etsy/phan at a forgiving level (quick, threshold=8)
and ran into various warnings. This fixes those.

Most of the changes are NFC (docblocks). The last two _ought_ to be
pretty safe, but another pair of eyes (and test run) would be good.

### CODE REVIEW

**Review by seamuslee001 — APPROVED** [review]

File: CRM/Core/Form.php [hunk 1]
```diff
@@ -67,16 +67,24 @@ class CRM_Core_Form extends HTML_QuickForm_Page {
...
-   * The options passed into this form
+   * (QUASI-PROTECTED) The options passed into this form
+   *
+   * This field should marked `protected` and is not generally
+   * intended for external callers, but some edge-cases do use it.
+   *
    * @var mixed
    */
-  protected $_options = NULL;
+  public $_options = NULL;
```
Comment (seamuslee001) [inline CRM/Core/Form.php]: @totten this change is generating a php error running api_report tests

**Comment by totten** [comment]
jenkins, test this please

File: CRM/Core/Form.php (hunk 1 above)
Comment (totten) [inline CRM/Core/Form.php]: Good catch.

IMHO, it's a defect if a subclass redeclares any of the parent's properties... but we'll work around...

**Comment by totten** [comment]
The reported failure `CRM_Contribute_Form_ContributionTest.testSubmitCreditCardPayPal` is pre-existing/not-relevant.

**Pull request merged**